packages/wallets/src/signers/non-custodial/ncs-evm-signer.ts

```diff
@@ -68,9 +68,4 @@ export class EvmEmailSigner extends EmailSigner {
             );
         }
     }
-
-    private static publicKeyToEvmAddress(publicKeyHex: string): string {
```

Comment:
why remove this?

Reply:
leftover private method from when the pregenerate signer was done client side